recipes/util-linux/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+./autogen.sh
+
+./configure
```

Comment:
Doesn't this need the `--prefix="${PREFIX}"`? Are there any other arguments we need?